client/src/selectors/eventLocationMapMarkers.js

```diff
@@ -0,0 +1,20 @@
+/*
+  Give it an array of event location objects and it will return and array of
+  react-native-maps/map-marker friendly objects (with some extra data mixed in)
+*/
+
+// eslint-disable-next-line import/prefer-default-export
+export const eventLocationMapMarkers = (eventLocations) => {
+  const mapMarkers = [];
+  eventLocations.forEach((r) => {
```

Comment:
Could this eventually be rewritten to something like:

`
const mapMarkers = eventLocations.filter(loc => (loc.locationLatitude !== null && loc.locationLongitude !== null).map(loc => ({ name, locationLatitude, locationLongitude, icon});
`